scripts/solr_builder/solr_builder/solr_builder_main.py

```diff
@@ -146,6 +199,9 @@ def _get_lite_metadata(ocaids, _recur_depth=0, _max_recur_depth=3):
         (and skip bad ocaids) 
         """
         if not ocaids or _recur_depth > _max_recur_depth:
+            logger.warning('Max recursion exceeded trying fetch IA data', extra={
+                'ocaids': ocaids
+            })
             return []
 
         r = requests.get("https://archive.org/advancedsearch.php", params={
```

Comment:
Sigh, this is actually blocking, even though it's in an async function. Fixed in a future PR, but too many conflicts.